README.md

```diff
@@ -20,9 +20,17 @@ When building to publish to npm the web project is actually injected into the co
 This is done under the hood by the `nx prepare core` step powered bythe buildCoreWithWeb/impl file.
 If changes are made to that file compile it `npx tsc tools/executors/buildCoreWithWeb/impl` via `npm run build:executor`. It may mention a couple tsc errors but confirm it compiles by running `git status` (the file `tools/executors/buildCoreWithWeb/impl.js` should be updated).
 
-# Publishing
+# Commiting
 
-Run `npm publish dist/libs/core --access public` from the root of the project.
+This repo uses conventional commits, an easy interactive way to stay compliant is to run `npm run commit` or `git cz` when committing.
```

Comment:
Will it prevent people from committing unless they use `git cz`? I don't see a pre commit hook. Would be nice to have all commits aligned. I guess I can test this after I merge

Reply:
@Gonzalo8642 it should be there unless it was only setup on my machine (it blocked me from committing using husky i believe).  Give it a try now that it's merged in and let me know